### ISSUE TITLE
Question about finetuning pipeline

### ISSUE THREAD

**Issue by greenyjwk** [microsoft/BiomedParse#105]
Hello, thank you for your great work.

I would like to ask a quick question about the segmentation outputs during the finetuning.

Under the 
/BiomedParse/modeling/architectures/seem_model_v1.py
GeneralizedSEEM class, forward_seg() func
In line 369. There is a line of code
` outputs = self.sem_seg_head.predictor(multi_scale_features, mask_features, extra=extra, task='seg')`
I have a quick question about the `outputs` 

Q) No matter the input image's ground truth is the outputs have the same when I track the `outputs`
I am working on two class segmentations. And what I thought and assumed was, depending on the text prompt(which is contained in the batched_inputs in line 367), the `outputs` should be different. However, no matter what the ground truth and text prompt are, the outputs look the same. I was wondering if it is fine.

If my question sounds vague, please feel free to let me know, so I can clarify it again.
Thank you.

**Comment by theodore-zhao** [comment]
Thanks for the question! Is this for the finetuned version? The extra variable contains the text prompt to guide the segmentation. The output of this part during inference is further selected by VL similarity: https://github.com/microsoft/BiomedParse/blob/main/modeling/architectures/seem_model_v1.py#L777 This part outputs the loss during training, which is further processed with https://github.com/microsoft/BiomedParse/blob/main/modeling/modules/criterion.py#L352 to provide different outputs. Hope this clarifies a bit!

**Comment by greenyjwk** [comment]
Thank you so much for sharing the source code link!
